src/vue/modules/buy-back-form/store/index.js

```diff
@@ -28,15 +26,15 @@ export const mutations = {
 
 export const actions = {
   async [types.LOAD_BALANCES] ({ commit, getters }) {
-    const endpoint = `/${HORIZON_VERSION_PREFIX}/accounts/${getters[types.accountId]}`
+    const endpoint = `/v3/accounts/${getters[types.accountId]}`
     const { data: account } = await api().getWithSignature(endpoint, {
       include: ['balances.state'],
     })
 
     commit(types.SET_BALANCES, account.balances)
   },
   async [types.LOAD_ASSETS] ({ commit, getters }) {
-    let response = await api().get(`${HORIZON_VERSION_PREFIX}/assets`)
+    let response = await api().get(`v3/assets`)
```

Comment:
let’s bring consistency. make sure all the api.get() and api.getWithSignature() are have starting slashes '/'